### ISSUE TITLE
Suggestion: Replace authorization on each request with inherited auth from parent collection

### ISSUE THREAD

**Issue by jkewley** [hkamel/azuredevops-postman-collections#8]
Each of these requests has the same Authorization setting - basic auth with the AccessToken password. To make it easier to add new requests to the collection, I'd suggest setting the Authorization at the collection level and having all requests inherit from the parent.

I'd be willing to do this if you're ok with it.

**Comment by hkamel** [comment]
I totally agree, and in fact I have done some of the APIs this way. The reason I kept it per API level is to help the user get all the details about running single API. 

But we can actually handle this on the documentation, thanks @jkewley
